packages/sqs/lib/sqs/AbstractSqsConsumer.ts

```diff
@@ -170,12 +172,21 @@ export abstract class AbstractSqsConsumer<
 
   public async start() {
     await this.init()
+    if (this.consumer) this.consumer.stop()
+
+    const visibilityTimeout = await getQueueAttributes(
+      this.sqsClient,
+      { queueUrl: this.queueUrl },
+      ['VisibilityTimeout'],
+    )
 
-    if (this.consumer) {
-      this.consumer.stop()
-    }
     this.consumer = Consumer.create({
+      sqs: this.sqsClient,
       queueUrl: this.queueUrl,
+      visibilityTimeout: visibilityTimeout.result?.attributes?.VisibilityTimeout
```

Comment:
🟢 it might read a bit better if we resolve visibilityTimeout inside a variable first, and then only pass here

Reply:
yeah 😓 sorry, will apply the change 🙇